Code/Editor/EditorViewportWidget.cpp

```diff
@@ -714,7 +620,7 @@ void EditorViewportWidget::OnEditorNotifyEvent(EEditorNotifyEvent event)
 
         if (m_renderViewport)
         {
-            m_renderViewport->SetInputProcessingEnabled(false);
+            m_renderViewport->GetControllerList()->SetEnabled(false);
```

Comment:
Almost missed this - this looks like it might be a merge artifact? `SetInputProcessingEnabled(false)` should be preferred here.

Reply:
Indeed, a merge artifact, address in 85459f1329